packages/client/test/integration/test-node/test/main.spec.ts

```diff
@@ -231,6 +261,13 @@ describe('Setting configuration', () => {
 
 describe('Blocks Stream API', () => {
```

Comment:
This test is useful, but let's put it in a separate file and don't mark it with allure for now, because this test tests the API, not the SDK.

Reply:
Move to the other file to create a special test file for API